mlos_bench/mlos_bench/tunables/tunable.py

```diff
@@ -32,7 +32,7 @@ class TunableDict(TypedDict, total=False):
     default: TunableValue
     values: Optional[List[Optional[str]]]
     range: Optional[Union[Sequence[int], Sequence[float]]]
-    special: Optional[Union[List[int], List[str]]]
```

Comment:
Hmm, I can't remember why I did it that way.  Wonder if that was just a mistake, or we really did have a good reason for it to be a `str` as well ...

Reply:
Wouldn't surprise me if there's some settings that are things like `[0, 1024]` or `AUTO`

Reply:
Yeah, sqlite has one of those for instance:
> The "PRAGMA cache_spill=N" form of this pragma sets a minimum cache size threshold required for spilling to occur. The number of pages in cache must exceed both the cache_spill threshold and the maximum cache size set by the [PRAGMA cache_size](https://www.sqlite.org/pragma.html#pragma_cache_size) statement in order for spilling to occur.
> The "PRAGMA cache_spill=boolean" form of this pragma applies across all databases attached to the database connection. But the "PRAGMA cache_spill=N" form of this statement only applies to the "main" schema or whatever other schema is specified as part of the statement.

https://www.sqlite.org/pragma.html#pragma_cache_spill

Reply:
Whereas, I think it would be unlikely for a `float` to be used for a special value (though maybe we shouldn't preclude that either).

Reply:
agree. Let's leave it like this for now, and I'll see how we can improve it later. Most likely, I'll revisit teh code when I address #635